src/ext/java/org/opentripplanner/ext/debug_client_geocoder/GeocoderResource.java

```diff
@@ -0,0 +1,155 @@
+package org.opentripplanner.ext.debug_client_geocoder;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import javax.ws.rs.DefaultValue;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import org.opentripplanner.api.mapping.FeedScopedIdMapper;
+import org.opentripplanner.model.StopLocation;
+import org.opentripplanner.routing.vertextype.StreetVertex;
+import org.opentripplanner.standalone.server.OTPServer;
+import org.opentripplanner.standalone.server.Router;
+
+/**
+ * OTP simple built-in geocoder used by the debug client.
+ */
+@Path("/routers/{ignoreRouterId}/geocode")
+@Produces(MediaType.APPLICATION_JSON)
+public class GeocoderResource {
+
+    /**
+     * @deprecated The support for multiple routers are removed from OTP2. See
+     * https://github.com/opentripplanner/OpenTripPlanner/issues/2760
+     */
+    @Deprecated
+    @PathParam("ignoreRouterId")
+    private String ignoreRouterId;
```

Comment:
I think for new endpoints we should not add deprecated params.

Reply:
+1 We only have these to be backward compatible, we should not add them to new endpoints.

Reply:
We discussed this at todays developer meeting and decided that we will remove the `\router\{routerId}` from all API paths in OTP 2.2 - THIS IS A BREAKING CHANGE!